lib/internal/Magento/Framework/DB/Ddl/Table.php

```diff
@@ -78,6 +78,20 @@ class Table
 
     const ACTION_SET_DEFAULT = 'SET DEFAULT';
 
+    /**#@+
+     * Option keys for column options
+     */
+    const OPTION_IDENTITY = 'identity';
+    const OPTION_UNSIGNED = 'unsigned';
+    const OPTION_NULLABLE = 'nullable';
+    const OPTION_PRIMARY = 'primary';
+    const OPTION_DEFAULT = 'default';
+    const OPTION_TYPE = 'type';
+    const OPTION_LENGTH = 'length';
+    const OPTION_SCALE = 'scale';
+    const OPTION_PRECISION = 'precision';
```

Comment:
Please sort them alphabetically (unless there is some special meaning in such order).

Reply:
Fixed